packages/core/src/solc_0.8.15/avatar/AvatarCollection.sol

```diff
@@ -456,7 +456,7 @@ contract AvatarCollection is
      * @dev reverts if not authorized
      * @custom:event {WaveSetup}
      */
-    function setPublicMint() external whenNotPaused authorizedRole(CONFIGURATOR_ROLE) {
+    function setPublicMint() external authorizedRole(CONFIGURATOR_ROLE) {
```

Comment:
bunch of whennotpaused removed, normal ?

Reply:
yes, we decided that only user-facing operations should be paused